nuget.config

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <clear />
+    <add key="repositoryPath" value="$\..\packages" />
+  </config>
+  <disabledPackageSources>
+    <clear />
+  </disabledPackageSources>
+  <packageSources>
+    <!-- For more info, see https://docs.nuget.org/consume/nuget-config-file -->
+    <clear />
+    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+
+    <!-- a local directory to allow testing nupkg files without pushing to a remote feed -->
+    <add key="packagestore" value="packagestore" />

```

Comment:
This will cause build to fail for those who don't have `packagestore` locally 🙂

Reply:
Good call.  I'll submit a PR with a fix.